pkgs/development/python-modules/syncserver/default.nix

```diff
@@ -16,18 +17,26 @@
 
 buildPythonPackage rec {
```

Comment:
My proposal: Move this out of `python-packages.nix` and use `buildPythonApplication` here. Then, you can use `packageOverrides` to specify the cornice version. In the NixOS module, you'll probably have to use `toPythonModule`.

Reply:
I'm sorry I'm confused. What should I move where ? Where to I use `packageOverrides` ? on `cornice` in each of the three modules that need version 0.17 ? what difference would using `buildPythonApplication` make ?
If you could either propose some changes or make a PR that would be super helpful and I'd be happy to test it.

Reply:
Syncserver should be called from `all-packages.nix` instead of `python-packages.nix`. We can also keep using `buildPythonPackage` if you add a comment that that's required for running it with paste.
For an example of how to use `packageOverrides`, have a look at `pkgs/servers/home-assistant/appdaemon.nix`.

Reply:
I need to change the cornice version on the three modules, otherwise I get a conflict

Reply:
Ohh, `packageOverrides` would work for the whole closure I see